backend/events/serializers.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import serializers
+
+from .models import Event, EventRegistration
+
+
+class EventSerializer(serializers.ModelSerializer):
```

Comment:
we can avoid `ModelSerializers` as they are slow this could be some discussion 

https://hakibenita.com/django-rest-framework-slow

Reply:
Am a bit busy with exams rn and was a bit unsure on how to make a custom serializer_class, other things are resolved so pushing them only for now, and adding this in an issue with other leftover tasks.

Reply:
Switched from `ModelSerializers` to regular serializers, please check if there's a way to build a completely custom serializer class (I was unsure how to do it).

Am a bit busy rn so pushing this only for now.

Reply:
Will add an issue for this